samples/samples/backup_sample.py

```diff
@@ -613,6 +697,55 @@ def copy_backup(instance_id, backup_id, source_backup_path):
 
 # [END spanner_copy_backup]
 
+# [START spanner_copy_backup_with_MR_CMEK]
+def copy_backup_with_multiple_kms_keys(
+    instance_id, backup_id, source_backup_path, kms_key_names
+):
+    """Copies a backup."""
+
+    from google.cloud.spanner_admin_database_v1.types import backup as backup_pb
+    from google.cloud.spanner_admin_database_v1 import CopyBackupEncryptionConfig
+
+    spanner_client = spanner.Client()
+    database_admin_api = spanner_client.database_admin_api
+
+    encryption_config = {
+        "encryption_type": CopyBackupEncryptionConfig.EncryptionType.CUSTOMER_MANAGED_ENCRYPTION,
+        "kms_key_names": kms_key_names,
+    }
+
+    # Create a backup object and wait for copy backup operation to complete.
+    expire_time = datetime.utcnow() + timedelta(days=14)
+    request = backup_pb.CopyBackupRequest(
+        parent=database_admin_api.instance_path(spanner_client.project, "hkw-nam3"),
```

Comment:
this might have been hardcoded for some testing, can we change it to the variable?

Reply:
Done.